src/pages/documentation/platform-assurance/tutorials/validation-summary.md

```diff
@@ -46,34 +41,33 @@ After you have tested your function, you can select **Upload** to save your vali
 
 | Key | Type | Description |
 | :--- | :--- | :--- | 
 | `message` | String | The validation message to display in the summary results. |
 | `events` | Array | An array of event uuids to be reported as matched or not matched. |
+| `links` | Array | An array of ValidationResultLink objects {( type: 'doc'|'product', url: String )} to reference documentation and other resources |
 | `result` | String | This is the validation result and is expected to be one of the enumerated strings: "matched", "not matched", "unknown" |
 
-![](./assets/validation-summary/invalid.png)
-
 ## Viewing Results
 
-The left panel displays the results of all enabled validations, including the tested function in the right panel.
+The results of the function will be displayed in the results section just below the code editor. When a validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, then those events will be highlighted in the timeline with the following colors:
+* Green - matched
+* Orange - unknown
+* Red - not matched
 
-When a validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, the left panel will render a link with the count of events that are not matched. Selecting the link will render a popover that contains a list of views. Selecting one of the views will navigate you to that view and will select those events not matched.
-
-![](./assets/validation-summary/results.png)
+![Timeling-Validation-Highlights-Screen-Shot](https://user-images.githubusercontent.com/6597105/198681412-93d10a5a-3212-4e85-850a-aeaf5caf0521.png)
 
 ## Troubleshooting
 
-If an error occurs in the JavaScript code editor, an `ERROR` status is displayed, and the reason for the error is output to the browser's developer console.
-
-Additionally, you can also reach out to our team on our Slack Channel at [https://projectgriffon.slack.com/](https://projectgriffon.slack.com/).
+You can add `console.log()` in your function to print items to the developer console. Alternatively, you can use the message property of the result object to debug messages to the results panel.
 
-To learn more about validations, you can visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/assurance-validation-plugins) GitHub. There you can find examples from validations owned by Adobe, as well as the [wiki](https://github.com/adobe/assurance-validation-plugins/wiki) that describes validations in more detail.
+If an error occurs in the JavaScript code editor, an error status is displayed along with the reason.
 
+To learn more about validations, you can visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/griffon-validation-plugins) GitHub. There you can find examples from validations owned by Adobe, as well as the [wiki](https://github.com/adobe/griffon-validation-plugins/wiki) that describes validations in more detail.
```

Comment:
Rephrase the last paragraph to: "To learn more about validations, please visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/griffon-validation-plugins) GitHub, where you will find examples of validations owned by Adobe. See the [wiki](https://github.com/adobe/griffon-validation-plugins/wiki) for more detailed descriptions of validations."